### PR TITLE
Add scripts used for reporting on my machine

### LINKED ISSUE #2050: Check for submissions in MECA_EXPORT_PENDING
We have had some instances of submissions not getting through to eJP and staff only realising when authors contact them requesting updates. These requests come often after a period of delay, during which the Editorial process has not even begun.

We have noticed that such submissions have become stuck in the MECA_EXPORT_PENDING state.

Can we alert staff and/or developers when a submission has been in this state for, e.g. >1 hour (and possibly retry the export automatically)?

Related to https://github.com/elifesciences/elife-xpub/issues/1950

/cc @diversemix

### PR DESCRIPTION
**Pull request by diversemix**
#### Background

Adds the scripts I use into git

#### Any relevant tickets

Closes #2050

#### How has this been tested?

Yes. tested locally

### CODE REVIEW

File: scripts/prod-status.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This script is designed to be called from 'prod-status-caller.sh'
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+HOST=52.90.128.135 # prod--xpub--1.elifesciences.org
```
Comment (diversemix) [inline scripts/prod-status.sh]: Waiting for the DNS entry to propagate.

**Review by petereast — APPROVED** [review]

File: scripts/prod-status.sh (hunk 1 above)
Comment (diversemix) [reply scripts/prod-status.sh]: Done

File: scripts/prod-status-caller.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [ $(hostname) != 'peter-xps' ]
+then
+  echo Should only be excuted on my PC for now!
```
Comment (giorgiosironi) [inline scripts/prod-status-caller.sh:5]: followed by `exit 1`?

Comment (diversemix) [reply scripts/prod-status-caller.sh:5]: Well spotted!

**Review by will-byrne — APPROVED** [review]

**Review by Cooryd — APPROVED** [review]

**Pull request merged**